nucypher/blockchain/eth/actors.py

```diff
@@ -1299,7 +1299,7 @@ def __init__(self,
                 self.stakes = StakeList(registry=self.registry, checksum_address=self.checksum_address)
                 self.stakes.refresh()
                 self.work_tracker = work_tracker or WorkTracker(worker=self)
-                self.work_tracker.start(act_now=False)
+                self.work_tracker.start(act_now=start_working_now)
```

Comment:
There's another invocation of `work_tracker.start()` as part of `Ursula.run()`. How does this interplay with the call in this line?
https://github.com/nucypher/nucypher/blob/523ea089f43581dda28e6b6a337a019d63a13e10/nucypher/characters/lawful.py#L1133-L1136